maintainers/maintainer-list.nix

```diff
@@ -5474,6 +5474,12 @@
     githubId = 136037;
     name = "Matthew Maurer";
   };
+  mausch = {
```

Comment:
This should be a separat commit. Please take a look into the contributions.md for the exact format.

Reply:
I'm already a maintainer via another package that was merged before this PR, so I've now removed this change from this PR.